app/uk/gov/hmrc/eusubsidycompliancefrontend/actions/ActionBuilders.scala

```diff
@@ -36,6 +39,18 @@ class ActionBuilders @Inject() (
   val notEnrolled: ActionBuilder[AuthenticatedRequest, AnyContent] = notEnrolledActionBuilder
 
   // GG Auth with ECC Enrolment and Verified Email Address
-  val verifiedEori: ActionBuilder[AuthenticatedEnrolledRequest, AnyContent] = verifiedEmailActionBuilder
+  val verifiedEori: ActionBuilder[AuthenticatedEnrolledRequest, AnyContent] = verifiedEoriActionBuilder
+
+  // GG Auth with ECC Enrolment and verify if payment has been submitted
+  val subsidyJourney: ActionBuilder[AuthenticatedEnrolledRequest, AnyContent] =
+    subsidyJourneyActionBuilder
+
+  // GG Auth with ECC Enrolment and verify if undertaking registration has been submitted
+  val enrolledUndertakingJourney: ActionBuilder[AuthenticatedEnrolledRequest, AnyContent] =
+    undertakingJourneySubmittedActionBuilder
+
+  // GG Auth with ECC Enrolment, check that they have a verified email and verify if undertaking registration has been submitted
+  val verifiedEoriUndertakingJourney: ActionBuilder[AuthenticatedEnrolledRequest, AnyContent] =
+    undertakingJourneyWithVerifiedEmailActionBuilder
 
```

Comment:
Is the purpose of ActionBuilders so that it can help build these separate journeys in both the SubsidyJourney and UndertakingJourney in the journeys file?